app/api/attendees.py

```diff
@@ -101,7 +102,13 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(TicketHolder)
+
+        if not (has_access('is_admin') or has_access('is_super_admin')):
+            order_subq = self.session.query(Order).\
+                         filter(or_(Order.status == 'completed', Order.status == 'placed')).subquery()
+            query_ = self.session.query(TicketHolder).join(order_subq, TicketHolder.order)
+        else:
+            query_ = self.session.query(TicketHolder)
```

Comment:
`query_ = self.session.query(TicketHolder)`

This section is being repeated. Remove the duplication